<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scopeWidget</class>
 <widget class="QWidget" name="scopeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scope Spectrum Correlation Statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="dabWaterfallAmplitude">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QwtPlot" name="dabWaterfall">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The waterfall of the spectrum of the incoming signal&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="scopeAmplification">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QwtPlot" name="dabScope">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Spectrumview. &lt;/p&gt;&lt;p&gt;The spectrum of the incoming
                      signal is shown here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                    </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QwtPlot" name="iqDisplay">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IQ constallation of the decoded signal. Ideally one sees 4
                          dots, one in each quadrant&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="scopeSlider">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zoom in/out&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
           <property name="sliderPosition">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Freq Correction</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLCDNumber" name="correctorDisplay">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Frequency correction, The software computed an estimate of the
                          frequency offset by lokking at phasedifferences in the samples in the time domain. Based on the difference found a
                          correction factor is applied to the input stream.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                        </string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>SNR</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLCDNumber" name="snrDisplay">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SNR, i.e. Signal Noise ratio, compoted from the difference in
                          signal strength between the datablocks and the nullperiod.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                        </string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Freq Offset</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLCDNumber" name="frequencyOffsetDisplay">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Frequeny offset, the computed frequency error, after applying
                          the estimated frequency correction.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                        </string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Time Offset</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLCDNumber" name="timeOffsetDisplay">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sample clock offset. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                        </string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Clock Offset</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLCDNumber" name="clockError">
           <property name="toolTip">
            <string>clock offset. The value indicates the number of samples too many or too few for 10 DAB frames (which is
                          about 1 second).
                        </string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Std. Deviation</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLCDNumber" name="quality_display">
           <property name="toolTip">
            <string>Quality indicator: the &quot;quality&quot; of the decoded signal is measured as Modulation Error Rate, using
                          the formula from section 6.9.2 of TR 101 290 Measurement guidelines for DVB.
                          An &quot;ideal&quot; (i.e. synthetic) signal reaches a value of over 20. Poor signals give values around 0
                        </string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Frequency</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLCDNumber" name="frequencyDisplay">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QCheckBox" name="cbLogIqScope">
           <property name="text">
            <string>IQ scope in log scale</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>OFDM Sym No</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLCDNumber" name="ofdmSymbNo">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Phase Correction</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLCDNumber" name="phaseCorrection">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="indexDisplay">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If more than one transmitter is detected, the relative distance - in
                  msec - is shown between the arrival time of the data of the transmitter that is used, and the arrival time of the data of
                  the other transmitter.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                </string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QwtPlot" name="impulseGrid">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Correlation display. Time synchronization is done by correlation.
                      The correlation result is depicted here. The selected &amp;quot;peak&amp;quot; is the one at (or near) sample 504.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                    </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
